src/plugins/discover/public/application/main/components/top_nav/open_alerts_popover.tsx

```diff
@@ -108,12 +108,7 @@ export function AlertsPopover({
       name: 'Alerting',
       items: [
         {
-          name: (
-            <>
-              {SearchThresholdAlertFlyout}
```

Comment:
This Flyout is already added on line 137.
Removing this to avoid multiple renders and redundant http requests.

Reply:
thx for fixing this! 👍